spread.yaml

```diff
@@ -356,6 +356,9 @@ backends:
             - ubuntu-22.04-64:
                   username: ubuntu
                   password: ubuntu
+            - ubuntu-22.10-64:
+                  username: ubuntu
+                  password: ubuntu
```

Comment:
won't this run all the tests there? why and do we really want to add this?

Reply:
this is only in the `qemu` section of spread.yaml - so is only relevant when running tests locally via qemu - I added this so that there was a relevant release to test this against at the time, but clearly this is now EOL so should be removed. I'll do that in a subsequent commit.

Reply:
Fixed in [506e224](https://github.com/snapcore/snapd/pull/12844/commits/506e224546ba8802b6a6f1ab62ba412209879138)